docs/src/man/basics.md

```diff
@@ -0,0 +1,2042 @@
+
+julia> s_german2 = @view german[end:-1:1, [1, 4]] # row and column subsetting
+1000×2 SubDataFrame
+  Row │ id     Job
+      │ Int64  Int64
+──────┼──────────────
+    1 │   999      2
+    2 │   998      2
+    3 │   997      2
+    4 │   996      3
+    5 │   995      1
+    6 │   994      2
+    7 │   993      3
+    8 │   992      1
+  ⋮   │   ⋮      ⋮
+  994 │     6      2
+  995 │     5      1
+  996 │     4      2
+  997 │     3      2
+  998 │     2      1
+  999 │     1      2
+ 1000 │     0      2
+     985 rows omitted
+```
+In the above example the number `2` and `5` represents the indices and as we know in Julia indices start from `1`.
+So, from `2:5` it will access the column from `:Age` to `:Housing`.
+
+```jldoctest dataframe
+julia> @view german[1:5, 1] # view the element type in vector form
+5-element view(::Vector{Int64}, 1:5) with eltype Int64:
+ 0
+ 1
+ 2
+ 3
+ 4
+
+julia> @view german[2, 2] # view of the number at 2nd row and 2nd column
+0-dimensional view(::Vector{Int64}, 2) with eltype Int64:
+22
+
+julia> @view german[3, 2:5] # a DataFrameRow, the same as for german[3, 2:5] without a view
+DataFrameRow
+ Row │ Age    Sex     Job    Housing
+     │ Int64  String  Int64  String
+─────┼───────────────────────────────
+   3 │    49  male        1  own
+```
+
+```julia
+julia> @time @view german[2:5, 2:5] # here you can see creation of view is very fast
+  0.000081 seconds (2 allocations: 112 bytes)
+4×4 SubDataFrame
+ Row │ Age    Sex     Job    Housing
+     │ Int64  String  Int64  String
+─────┼───────────────────────────────
+   1 │    22  female      2  own
+   2 │    49  male        1  own
+   3 │    45  male        2  free
+   4 │    53  male        2  free
+```
+
+## Using `combine`, `select`, `select!`, `transform`, and `transform!`
+
+In DataFrames.jl we have five functions that we can use to perform transformations of columns 
+of a data frame:
+- `combine`: create a new data frame populated with columns that are results of transformation 
+  applied to the source data frame columns;
+- `select`: create a new data frame that has the same number of rows as the source data frame 
+  populated with columns that are results of transformations applied to the source data frame 
+  columns; (the exception to the above number of rows invariant is `select(german)` which produces
+  an empty data frame);
+- `select!`: the same as `select` but updates the passed data frame in place;
+- `transform`: the same as `select` but keeps the columns that were already present in the data frame
+  (note though that these columns can be potentially modified by the transformation passed to `transform`);
+- `transform!`: the same as `transform` but updates the passed data frame in place.
+
+You can also use the `select` and `select!` functions to select, rename, and transform columns in a data frame.
+The simplest way to specify a transformation is:
+
+- `source_column => transformation => target_column_name`
```

Comment:
how does this look after rendering? maybe give also some concrete examples and explain that e.g. `transformation` is a function

Reply:
I did not get this, sorry.

Reply:
> how does this look after rendering?

I am asking how does this part of documentation look after generation of HTML files. I feel that it will be incorrectly formatted.

> maybe give also some concrete examples and explain that e.g. `transformation` is a function

In your description now it is not clear what are valid values of `tranfromation` argument.

Reply:
> In your description now it is not clear what are valid values of the `transformation` argument.
Yeah, I have fixed it but I am not able to verify after rendering because I am building the docs but because of some previous cache it's still showing previous errors. I tried to clear those errors but again same things are happening.